TwilioVerifySDK/TwilioSecurity/Sources/Storage/SecureStorage.swift

```diff
@@ -125,4 +129,27 @@ extension SecureStorage: SecureStorageProvider {
       }
     }
   }
+
+  private func retry<T>(
+    tries: Int = 3,
+    block: () throws -> T
```

Comment:
Should we include a [delay](https://github.com/twilio/twilio-verify-android/blob/main/security/src/main/java/com/twilio/security/crypto/AndroidKeyManager.kt#L143) to give some time?